src/rpc/Errors.h

```diff
@@ -17,6 +17,7 @@ namespace RPC {
  */
 enum class ClioError {
     rpcMALFORMED_CURRENCY = 5000,
+    rpcMALFORMED_ADDRESS = 5003,
```

Comment:
Note that we skip to 5003. This is because in PR #417 we add 5001 and 5002 and this will be merged after #417